MAJ Passenger/nginx + mongodb + readme

FloGro opened the pull request:
Mise-à-jour de Webrick vers Passenger
Mise-à-jour de la base de données SQLite3 vers Mongodb
/!\ Faire un aptitude install mongodb mongodb-server mongodb-dev
/!\ Puis faire un bundle install ou bundle update
Mise-à-jour du README

FloGro committed:
MAJ Passenger et Mongodb

Mise-à-jour de Webrick vers Passenger
Mise-à-jour de la base de données SQLite3 vers Mongodb
/!\ Faire un aptitude install mongodb mongodb-server mongodb-dev
/!\ Puis faire un bundle install ou bundle update

FloGro committed:
Mise-à-jour du README

Description des paquets à installer
Installation des gems
Lancement
Arrêt

FloGro committed:
Update README.rdoc

FloGro committed:
Mise-en-page README

jbraffin commented on README.rdoc:
```diff
@@ -1,28 +1,47 @@
+* passenger start -p 80 -d
+
+Arrêt :
+* passenger stop -p 80
```
Vraiment cool la doc flo ^^

jbraffin commented on rails-bdd.sh:
```diff
@@ -15,7 +13,7 @@ b=0
+	 r)
              r=1
              ;;
 	 s)
```
Est ce que tu pourrais juste m'expliquer le 6 dernières lignes 0o ?

FloGro replied:
Les 6 dernières lignes ? Celles qui envoient les seeds quand on fait -s ?

jbraffin replied:
je comprends pas la syntaxe du bloque case que j'ai commenté

FloGro replied:
Ah ok. T'as 3 options r,s,b, qui ne prennent pas d'argument, je me sers des $r,s,b=0 | 1 comme drapeau pour le reste du script. r pour reset, s pour seeds, b pour bundle exec.

FloGro committed:
migration vers mongodb

FloGro commented:
Nouveau commit ajouté ! 
Migration vers mongodb des librairies, sachant que la base mongodb est indépendante du projet, pas la peine de la régénérer à chaque rebase, en revanche, lancer une fois : deployment/init_mongodb_lexiques.sh

FloGro committed:
Correctif : Texte-à-trou

Texte-à-trou, lorsque l'on entrait un mot n'étant pas reconnu dans le
dictionnaire avec le seuil d'erreur à 1 ou à 2, cela générait une
erreur. Mais ça c'était avant :)

FloGro commented:
Encore un correctif ajouté à la pull request, histoire d'avoir un truc stable pour Janvier :)

thomaschauffour commented:
Fabuleux !

Anne-So commented:
=D

FloGro commented:
Bon je merge alors !
N'oubliez pas : 
git rebase
aptitude install verbiste mongodb mongodb-server mongodb-dev
deployment/init_mongodb_lexique.sh
bundle install
et passenger start -p 80 [-d] pour le lancer !
